include/lsst/afw/table/Exposure.h

```diff
@@ -381,7 +381,7 @@ class ExposureCatalogT : public SortedCatalogT<RecordT> {
      *
      *  The returned array's records are shallow copies, and hence will not in general be contiguous.
      */
-    ExposureCatalogT<RecordT> subset(ndarray::Array<bool const,1> const & mask) const {
+    ExposureCatalogT subset(ndarray::Array<bool const,1> const & mask) const {
```

Comment:
I'd rather not merge these kinds of changes into the pybind11 branch when not really needed. Perhaps this falls into the category of one-line fixes that don't need ticketing and review? If so, then rather push it to master and we'll pick it up with the next rebase.